packages/sdk/src/browser/index.ts

```diff
@@ -51,8 +83,10 @@ class ConnectError extends Error {}
 type StartConnectOpts = {
   /**
    * The token used for authenticating the connection.
+   *
+   * Optional if client already initialized with token
    */
-  token: string;
+  token?: string;
```

Comment:
Do we want to remove this and require the token be configured when the client is initialized?

This would be a breaking change.